x-pack/plugin/enrich/src/test/java/org/elasticsearch/xpack/enrich/EnrichMultiNodeIT.java

```diff
@@ -128,8 +128,8 @@ private static void enrich(List<String> keys, String coordinatingNode) {
 
         for (int i = 0; i < numDocs; i++) {
             GetResponse getResponse = client().get(new GetRequest("my-index", Integer.toString(i))).actionGet();
-            Map<String, Object> source = getResponse.getSourceAsMap();
-            assertThat(source.size(), equalTo(1 + DECORATE_FIELDS.length));
+            Map<?, ?> source = (Map<?, ?>) getResponse.getSourceAsMap().get("user");
```

Comment:
curious why `<String,Object>` --> `<?,?>`

Reply:
To avoid unchecked casts, otherwise we need to add a suppress annotation.
(the source is not direct source, but the content under the `user` key)